helper/src/main/java/me/lucko/helper/utils/XMaterial.java

```diff
@@ -0,0 +1,1686 @@
+        if (current == MinecraftVersion.UNKNOWN) return true;
+
+        int ver = Integer.parseInt(StringUtils.replace(StringUtils.replace(version.name(), "VERSION_", ""), "_", ""));
+        int currentVer = Integer.parseInt(StringUtils.replace(StringUtils.replace(current.name(), "VERSION_", ""), "_", ""));
+
+        return currentVer >= ver;
+    }
+
+    /**
+     * Converts the material's name to a string with the first letter uppercase.
+     *
+     * @return a converted string.
+     */
+    @Nonnull
+    public static String toWord(@Nonnull Material material) {
+        return toWord(material.name());
+    }
+
+    /**
+     * Parses a material name like a normal like.
+     * Normal names have underlines removed and each word starts with caps.
+     *
+     * @param name the name of the material.
+     * @return a cleaned material name.
+     */
+    @Nonnull
+    private static String toWord(@Nonnull String name) {
+        StringBuilder translated = new StringBuilder();
+        String[] separator = name.split("_");
+
+        if (separator.length == 0) {
+            translated.append(name.substring(1).toLowerCase());
+            return translated.toString();
+        }
+
+        Arrays.stream(separator).forEach(s -> translated.append(s.charAt(0)).append(s.substring(1).toLowerCase()).append(" "));
+        return translated.toString();
+    }
+
+    /**
+     * Compares two major versions. The current server version and the given version.
+     *
+     * @param version the version to check.
+     * @return true if is the same as the current version or higher.
+     */
+    public static boolean isVersionOrHigher(@Nonnull String version) {
+        int currentVer = Integer.parseInt(getExactMajorVersion(StringUtils.replace(Bukkit.getVersion(), ".", "")));
+        int versionNumber = Integer.parseInt(StringUtils.replace(version, ".", ""));
+
+        return currentVer >= versionNumber;
+    }
+
+    /**
+     * Gets the exact major version (..., 1.9, 1.10, ..., 1.14)
+     *
+     * @param version Supports {@link Bukkit#getVersion()}, {@link Bukkit#getBukkitVersion()} and normal formats such as "1.14"
+     * @return the exact major version.
+     */
+    @Nonnull
+    public static String getExactMajorVersion(@Nonnull String version) {
+        // getBukkitVersion()
+        if (version.contains("SNAPSHOT") || version.contains("-R"))
+            version = version.substring(0, version.indexOf("-"));
+        // getVersion()
+        if (version.contains("git"))
+            version = StringUtils.replace(version.substring(version.indexOf("MC:") + 4), ")", "");
+        if (version.split(Pattern.quote(".")).length > 2) version = version.substring(0, version.lastIndexOf("."));
+        return version;
+    }
+
+    /**
+     * Parses the string arugment to a version.
+     * Supports {@link Bukkit#getVersion()}, {@link Bukkit#getBukkitVersion()} and normal formats such as "1.14"
```

Comment:
Additional paragraphs in comments should include an empty line between previous paragraphs.